common/browser.go

```diff
@@ -486,6 +486,16 @@ func (b *Browser) Close() {
 	b.conn.Close()
 }
 
+// CloseContext i a short-cut function to close the current browser's context.
+// If there is not an active browser context, it returns an error.
```

Comment:
```suggestion
// CloseContext is a short-cut function to close the current browser's context.
// If there is no active browser context, it returns an error.
```

Reply:
Fixed in [4f2bd5c](https://github.com/grafana/xk6-browser/pull/1074/commits/4f2bd5c5c967a8b52c36ebbad9eb9dcab9039c91).